docs/docsource/operation/account_operations/manage_access_keys.rst

```diff
@@ -0,0 +1,51 @@
+.. _manage_access_keys:
+
+Manage Access Keys
+==================
+
+Creating Access Keys Using the UI
+---------------------------------
+
+#. Log into the |product| UI using previously provisioned credentials.
+
+   .. image:: ../../graphics/sign_into_artesca.PNG
+
+#. Select the **Accounts** tab at the top of the screen.
+
+   .. image:: ../../graphics/account_banner.PNG
+
+#. Select blue button  **+Create Access key**
+
+   .. image:: ../../graphics/accounts-page.PNG
+
+#. Select **Continue** on the **Create Root user Access keys** screen.
+
+   .. image:: ../../graphics/root_user_access_keys_dialog.PNG
+
+#. Save the Secret Access key ID. 
```

Comment:
Key (capital letter) + maybe add "by clicking the copy button, and save it in a secure location." (then get rid of the sentence below)